.github/actions/verify-submodule-pull-request/Check-Parent.ps1

```diff
@@ -8,9 +8,8 @@ if (!($Branch -match '(\w+-\d+)'))
     exit
 }
 
-$url = "https://api.github.com/repos/$Repository/pulls?state=open&per_page=100"
-$response = Invoke-WebRequest $url -Headers (Get-GitHubApiAuthorizationHeader) -Method Get
-$titles = $response.Content | ConvertFrom-Json | ForEach-Object { $PSItem.title }
+# See https://cli.github.com/manual/gh_pr_list and https://cli.github.com/manual/gh_help_formatting
+$titles = gh pr list --repo $Repository --limit 100 --json title --template '{{range .}}{{tablerow .title}}{{end}}'
```

Comment:
Will this list the 100 most recent open PRs? (So, not issue if we have more than 100 PRs altogether.)

Reply:
Yes, it only shows open PRs by default (if you don't include the `--state` switch).

Reply:
OK then!